lib/cluster.js

```diff
@@ -31,14 +31,12 @@ function Worker(options) {
   this.exitedAfterDisconnect = undefined;
 
   Object.defineProperty(this, 'suicide', {
-    get: internalUtil.deprecate(() => {
```

Comment:
OK, I missed that, the diff is hard to read. Shouldn't a change to lib/cluster.js have `cluster: ` in its commit message? Even if its just a stylistic change?